tests/README.md

```diff
@@ -9,3 +9,8 @@ where:
 - `-failfast` makes the test stop after first failure
 - `-count 1` is to avoid Go test caching for example when testing a rebuilt snap
 
+## Environment variables 
+
+Some environment variables can modify the test functionality. Refer to these in
+[the documentation](https://pkg.go.dev/github.com/canonical/matter-snap-testing@v1.0.0-beta.3/env)
```

Comment:
It is going to be hard to maintain this URL, because of the version.

Since we don't have a stable release yet, it may not be possible to provide a link with the `latest` alias. If that is true, I suggest giving a link to https://pkg.go.dev/github.com/canonical/matter-snap-testing and pointing the reader to the env package.

Reply:
There is a way to use a version agnostic link removing the `@v1.0.0-beta.3` then it will always refer to the `latest` one.